README.md

```diff
@@ -18,6 +16,16 @@ TODO: Add a summary of who the application is for and what it will do.
 
   Our intention is that the example should include enough to get the application started quickly. If this is not the case, please ask another developer for a copy of their `/.env.development.local` file.
 
+  ```bash
```

Comment:
This looks a bit confused now as the changes to the README on master are not reflected

Reply:
I see. We're looking to only have your new addition showing here at the moment. When it is ready, we can `git rebase origin/develop` to pull in my changes to the README. Then it should make sense.

Reply:
Right, this PR only has the bit I added now 👍

Reply:
Looks good. Now the rebase should add my bits as existing changes, and it should ask you to resolve the conflict and slot it into the right place.